libqtile/widget/volume.py

```diff
@@ -72,13 +68,17 @@ class Volume(base._TextBox):
     ]
 
     def __init__(self, **config):
-        base._TextBox.__init__(self, '0', width=bar.CALCULATED, **config)
+        base._TextBox.__init__(self, '      ', width=bar.CALCULATED, **config)
+        self.surfaces = {}
+        self.muted = False
+        self.wasmuted = False
+        # self.volume = 0
+        # self.update()
         self.add_defaults(Volume.defaults)
         if self.theme_path:
             self.length_type = bar.STATIC
             self.length = 0
-        self.surfaces = {}
-        self.volume = None
+        self.volume = 0
```

Comment:
I think this should be None, so we always do the drawing update on the first update